test/Be.Vlaanderen.Basisregisters.Shaperon.Tests/DbaseIntegerDigitsTests.cs

```diff
@@ -74,14 +74,22 @@ public void ToInt32ReturnsExpectedValue()
         [Fact]
         public void ToLengthReturnsExpectedValue()
         {
-            var value = _fixture.Create<int>().AsDbaseIntegerDigitsValue();
+            var value = _fixture.Create<int>().AsDbaseFieldCompatibleDbaseIntegerDigitsValue();
```

Comment:
This test used to sporadically fail because a `DbaseIntegerDigits(0)` can not be converted to a `DbaseFieldLength` (requires value between 1 and 254).